src/Tribe/Assets_Pipeline.php

```diff
@@ -35,9 +35,9 @@ public function prevent_underscore_conflict( $tag, $handle ) {
 
 		if ( 'underscore' === $handle ) {
 			$dir = Tribe__Main::instance()->plugin_url . 'src/resources/js';
-			$tag = "<script src='{$dir}/underscore-before.js'></script>\n"
+			$tag = "<script src='{$dir}/underscore-before.js' defer></script>\n"
 			       . $tag
-			       . "<script src='{$dir}/underscore-after.js'></script>\n";
+			       . "<script src='{$dir}/underscore-after.js' defer></script defer>\n";
```

Comment:
@adamsilverstein I wasn't expecting `</script defer>` is that intentional?

Reply:
no that looks off, will fix - thanks.

Reply:
fixed